ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Query/Version.hs

```diff
@@ -11,13 +11,8 @@ import           Ouroboros.Network.NodeToClient.Version
 -- Multiple top level queries are now supported. The encoding now has
 -- constructor tags for the different top level queries for QueryVersion1 onwards.
 data QueryVersion
-    -- | Only the 'BlockQuery' constructor of 'Query' is supported. The binary
```

Comment:
Note to self: the `"The binary encoding is backwards compatible: it does not introduce any constructor tag around the 'BlockQuery'."` sentence was specific to that version, so we need not retain it.